rollup/cmd/rollup_relayer/app/app.go

```diff
@@ -78,12 +79,17 @@ func action(ctx *cli.Context) error {
 		log.Crit("failed to create l2 relayer", "config file", cfgFile, "error", err)
 	}
 
-	chunkProposer := watcher.NewChunkProposer(subCtx, cfg.L2Config.ChunkProposerConfig, db, registry)
+	network := network.Network(ctx.String(utils.NetworkFlag.Name))
+	if !network.IsKnown() {
```

Comment:
Is there a need to use arbitrary genesis configurations in rollup relayer or just relying on the hard coded configurations from l2geth is fine? cc: @Thegaram

Reply:
walkthrough the code, if don't load the genesis.json, i think the block_number is all 0

Reply:
> if we can support custom configurations

in that case, I will remove the `Network` enum and just pass in path to the `genesis.json` to be parsed by `rollup_relayer`. does it make sense to you @georgehao  @Thegaram ?